src/backend/routers/para.ts

```diff
@@ -54,5 +69,41 @@ export const para = router({
       return result;
     }),
 
+  /**
+   * The function called in the frontend.
+   * Checks if para exists and finds it. If it doesn't we create one using createPara(). Then we assign the para to CM.
+   * Sends email only if createPara() is called. Not sure if this needs to be changed.
+   */
+  assignParaToCaseManager: authenticatedProcedure
```

Comment:
how will this be used in the UI? I'm thinking we might want to reduce the scope of this to assign a para to a case manager given an email or para ID and remove the creation logic to make it a little easier to understand

Reply:
Agree. Ideally the signature of this method should be:
assignParanToCaseManager(para_id, case_manager_id)

The para creation can be handled in the function that calls it. This is the single responsibility principle of functions.